build.psm1

```diff
@@ -446,6 +446,32 @@ Fix steps:
         Pop-Location
```

Comment:
@SteveL-MSFT, your last commit had 10 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Remote import-module tests.Import-Module can import over remote session: pssession
```powershell
Expected exactly 'TestImport', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\RemoteImportModule.Tests.ps1: line 105
105:             $module.Name | Should -BeExactly "TestImport"
```
Remote import-module tests.Import-Module can import over remote session: minimumversion
```powershell
Expected exactly 'TestImport', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\RemoteImportModule.Tests.ps1: line 105
105:             $module.Name | Should -BeExactly "TestImport"
```
Remote import-module tests.Import-Module can import over remote session: requiredversion
```powershell
Expected exactly 'TestImport', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\RemoteImportModule.Tests.ps1: line 105
105:             $module.Name | Should -BeExactly "TestImport"
```
Remote import-module tests.Import-Module can import over remote session: maxiumversion
```powershell
Expected exactly 'TestImport', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\RemoteImportModule.Tests.ps1: line 105
105:             $module.Name | Should -BeExactly "TestImport"
```
Remote import-module tests.Import-Module can import over remote session: fullyqualifiedname
```powershell
Expected exactly 'TestImport', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\RemoteImportModule.Tests.ps1: line 105
105:             $module.Name | Should -BeExactly "TestImport"
```